deploy/docker/Dockerfile.msodbc

```
@@ -7,15 +7,14 @@
 ## $ docker build -f=Dockerfile.msodbc -t emqx-enterprise-with-msodbc:5.0.3-alpha.2 .
 ## ```
 
-# FROM emqx/emqx-enterprise:latest
-FROM emqx/emqx-enterprise:5.0.3-alpha.2
+FROM emqx/emqx-enterprise:latest
```

```suggestion
ARG VERSION=5.2.0
FROM emqx/emqx-enterprise:$VERSION
```
I think `latest` is not good, we should use more explicit tags.